source/Options.md

```diff
@@ -0,0 +1,115 @@
+# Options
+
+
+## \--mode
+
+This option allows you to set up the read/write mode for PXC cluster
+nodes in the ProxySQL database based on the hostgroup. The only
+supported modes are`singlewrite` and `loadbal`.
+
+The `singlewrite` option is the default mode. Percona XtraDB Cluster only accepts writes on a single node. The write node may accept read requests, depending on the `--writers-are-readers` value. The other nodes are read-only and accept only read statements.
+
+The `--write-node` option controls which node ProxySQL uses as
+the writer node. The writer node is specified as the  If `--write-node` is used, the writer node has a weight of **1000000**, which is greater than the default weight of **1000**, and the node has the assigned address:port of -**10.0.0.51:3306**.
```

Comment:
The sentence "The writer node..." has no ending.

Reply:
Fixed.